io.github.archisman_panigrahi.typhoon.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.archisman_panigrahi.typhoon</id>
+  <name>Typhoon</name>
+  <summary>Stylish weather application</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <description>
+    <p>
+      Typhoon is a beautiful weather application powered by Open-Meteo and OpenStreetMap that provides real-time weather updates and forecasts. It is based on Stormcloud.
+    </p>
+  </description>
+  <url type="homepage">https://archisman-panigrahi.github.io/typhoon</url>
+  <url type="bugtracker">https://github.com/archisman-panigrahi/typhoon/issues</url>
+  <developer id="io.github.archisman-panigrahi">
+    <name>Archisman Panigrahi</name>
+  </developer>
+  <project_license>GPL-3.0-or-later</project_license>
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+  <launchable type="desktop-id">io.github.archisman_panigrahi.typhoon.desktop</launchable>
+  <categories>
+    <category>Utility</category>
+    <category>Network</category>
+  </categories>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Main window showing weather forecast</caption>
+      <image>https://github.com/archisman-panigrahi/typhoon/raw/master/assets/screenshots/typhoon.png</image>
```

Comment:
screenshots URL should never reference the tip of a branch. It's not immutable.

Reply:
fixed in https://github.com/flathub/flathub/pull/6441/commits/57e574e1b9b397258fde460cbbb575aa9224c004